src/schema/v2/artwork/index.ts

```diff
@@ -749,15 +749,36 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
       },
       artaShippingEnabled: {
         type: GraphQLBoolean,
+        deprecationReason: deprecate({
+          inVersion: 2,
+          preferUsageOf: "processWithArtsyShippingDomestic",
+        }),
         resolve: ({ arta_enabled }) => arta_enabled,
       },
       artsyShippingInternational: {
         type: GraphQLBoolean,
         resolve: ({ artsy_shipping_international }) =>
           artsy_shipping_international,
       },
+      processWithArtsyShippingDomestic: {
+        type: GraphQLBoolean,
+        description:
+          "Returns true if this work is eligible to be automatically opted into Artsy Domestic Shipping",
+        resolve: (artwork) => {
+          return Boolean(
+            artwork.process_with_artsy_shipping_domestic ||
```

Comment:
Documented in subtask, will be doing a cleanup pass to remove these `||`s after force and gravity (removal of process_with_arta_shipping from json fields) are deployed